service_configuration_lib/spark_config.py

```diff
@@ -1061,15 +970,11 @@ def compute_approx_hourly_cost_dollars(spark_conf, paasta_cluster, paasta_pool):
             spark_conf.get('spark.dynamicAllocation.minExecutors', min_cores),
         ))
 
-    if paasta_pool == 'batch':
-        default_cost_factor = 0.041
-    else:
-        default_cost_factor = 0.142
-    cost_factor = spark_costs.get(paasta_cluster, dict()).get(paasta_pool, default_cost_factor)
+    cost_factor = spark_costs.get(paasta_cluster, dict())[paasta_pool]
 
     min_dollars = round(min_cores * cost_factor, 5)
     max_dollars = round(max_cores * cost_factor, 5)
-    if max_dollars * 24 > spark_constants.get('high_cost_threshold_daily', 500):
+    if max_dollars * 24 > spark_constants['high_cost_threshold_daily']:
```

Comment:
Nit: Define constants at the top.
```suggestion
    if max_dollars * HIGH_COST_WARNING_THRESHOLD > spark_constants['high_cost_threshold_daily']:
```